src/main/daml/Daml/Finance/Claims/Util/Builders.daml

```diff
@@ -169,15 +169,15 @@ createFxAdjustedPrincipalClaim dateToTime ownerReceives fxRateMultiplier notiona
 
 -- | Create the claim for a long vanilla option (cash-settled, automatically exercised).
 createVanillaOptionClaim : (Date -> Time) -> Decimal -> Observable -> Deliverable -> Date -> Bool
-  -> Bool -> Decimal -> C
+  -> Optional Decimal -> Decimal -> C
 createVanillaOptionClaim dateToTime strike spot cashInstrument expiryDate isCall
-  percentageOfStrike notionalMultiplier =
+  referenceLevel notionalMultiplier =
   let
     intrinsicValueAbs = if isCall then Observe spot - Const strike
                                   else Const strike - Observe spot
-    intrinsicValue = if percentageOfStrike
-      then intrinsicValueAbs / Const strike
-      else intrinsicValueAbs
+    intrinsicValue = case referenceLevel of
+      Some ref -> intrinsicValueAbs / Const ref
```

Comment:
Here we are dividing the strike by the ref level, which means that the input strike must be provided in absolute terms. 

A BRC is often structured using a relative strike (e.g. 70 % or 100 % of ref level). 

I would suggest to account for that in the builder and use `intrinsicValue = spot / ref level - strike` rather than `(spot - strike) / ref level`

Reply:
Currently, all instruments that use this builder have absolute strikes (both the options and the BRC). 

I think I would prefer keeping it as it is, until we start using relative strikes as well. At that point, we can extend this builder, or split it up into two different ones.

Reply:
Sure. I think the BRC should be setup using relative strikes, but no strong opinion on when this should be happen

Reply:
Here is an issue to track this: https://github.com/digital-asset/daml-finance/issues/887